test/bigtest/interactors/location-institutions.js

```diff
@@ -0,0 +1,14 @@
+import {
+  interactor,
+  scoped,
+} from '@bigtest/interactor';
+
+import ButtonInteractor from '@folio/stripes-components/lib/Button/tests/interactor'; // eslint-disable-line
+
+@interactor class InstitutionsInteractor {
+editRow = scoped('[class*="editListRow"]');
```

Comment:
It looks like there are not spaces before variables declarations. It can be applied for lines 9, 10, 11